src/org/thoughtcrime/securesms/connect/KeepAliveService.java

```diff
@@ -123,8 +123,8 @@ static private void createFgNotificationChannel(Context context) {
         if(!ch_created) {
             ch_created = true;
             NotificationChannel channel = new NotificationChannel(NotificationCenter.CH_PERMANENT,
-                "Receive messages in background.", NotificationManager.IMPORTANCE_MIN); // IMPORTANCE_DEFAULT will play a sound
-            channel.setDescription("Ensure reliable message receiving.");
+                "Backgrund connection. This can safely be turned off.", NotificationManager.IMPORTANCE_MIN); // IMPORTANCE_DEFAULT will play a sound
```

Comment:
Telling the user that something can be "safely turned off" raises the question why its there in the first place.

So if we don't need it: Then we should remove it
If we only need it on special situations (old Androids?): Only use it there (and same wording as below).
We need it to display messages quickly: "Background connection. Needed to display new messages quickly."
We need it to display messages at all when DC is not in the foreground: "Background connection. Needed to inform you of new messages."

(We also should really put the strings to the strings.xml so they are translated)